extension/.nycrc.json

```diff
@@ -0,0 +1,27 @@
+{
+  "extends": "@istanbuljs/nyc-config-typescript",
+  "require": [
+    "ts-node/register",
+    "source-map-support/register"
+  ],
+  "include": [
+    "src/**/*.ts"
+  ],
+  "exclude": [
+    "src/test/*",
+    "out/test/*",
```

Comment:
This one shouldn't be needed, right? I mean, it's not in the `include` filter...